sdk/servicebus/service-bus/tslint.json

```diff
@@ -30,6 +30,12 @@
         "variable-declaration": "onespace"
       }
     ],
+    "import-blacklist": [
+      true,
+      {
+        "rhea/typings/types": ["Typed"]
```

Comment:
The idea is to avoid `import` and `require` statements from `rhea`. Anything that is needed from `rhea` should be made available from `rhea-promise`.  This helps us avoid situations where multiple versions of `rhea` could be installed in node_modules.

The definition and examples provided for [import-blacklist](https://palantir.github.io/tslint/rules/import-blacklist/) do support *regular expressions*.
```json5
"import-blacklist": [true, "rxjs", {"lodash": ["pull", "pullAll"]}, [".*\\.temp$", ".*\\.tmp$"]]
```
We could block
- anything imported from `rhea/typings/*`
```json5
"import-blacklist": [true, ["rhea\/typings.*$"]]
```
- anything imported from `rhea` (no regex here). However, I am not sure how good tslint is in differentiating between "rhea" and "rhea-promise". Hope it does the right thing.
```json5
"import-blacklist": [true, "rhea"]
```

Reply:
Got it, I was about to send an update considering regex-es rhea\* and rhea :)

Reply:
I've updated the rule.
Verified that this disallows both rhea and rhea\typings\* while allowing rhea-promise

Once the rhea-promise changes are merged, these changes can be merged in as well
(Verified by pointing project to my rhea-promise git branch)

cc @amarzavery @ramya-rao-a 
(Looks like Ramya doesn't have permissions to merge this PR)